chemprop/cli/main.py

```diff
@@ -28,30 +28,31 @@ def main():
 
     parent = ArgumentParser(add_help=False)
     parent.add_argument(
-        "--logfile",
-        "--log",
-        nargs="?",
-        default=DEFAULT_LOGFILE,
-        help=f"the path to which the log file should be written. Specifying just the flag (i.e., '--log/--logfile') will automatically log to a file '{LOG_DIR}/MODE/{NOW}.log', where 'MODE' is the CLI mode chosen.",
+        "--output-dir",
+        "--save-dir",
+        help="Directory where outputs will be saved. Defaults to <current working directory>/<mode>/<date time>.",
+    )
+    parent.add_argument(
+        "-q", "--quiet", action="count", default=0, help="supression level for logging"
     )
-    parent.add_argument("-v", "--verbose", action="count", default=0, help="the verbosity level")
 
     parents = [parent]
     for subcommand in SUBCOMMANDS:
         subcommand.add(subparsers, parents)
 
     args = parser.parse_args()
-    logfile, verbose, mode, func = (
-        pop_attr(args, attr) for attr in ["logfile", "verbose", "mode", "func"]
-    )
+    quiet, mode, func = (pop_attr(args, attr) for attr in ["quiet", "mode", "func"])
 
-    logfile = Path(LOG_DIR / mode / f"{NOW}.log" if logfile is None else logfile)
-    logfile.parent.mkdir(parents=True, exist_ok=True)
+    args.output_dir = (
+        Path.cwd() / "output_files" / mode if args.output_dir is None else args.output_dir
+    ) / NOW
+    args.output_dir.mkdir(parents=True, exist_ok=True)
+    logfile = args.output_dir / "chemprop.log"
 
     logging.basicConfig(
         filename=str(logfile),
         format="%(asctime)s - %(levelname)s:%(name)s - %(message)s",
-        level=LOG_LEVELS[min(verbose, len(LOG_LEVELS) - 1)],
+        level=LOG_LEVELS[max(0, len(LOG_LEVELS) - 1 - quiet)],
```

Comment:
This sets the default logging to `logging.DEBUG`. A better default could be `logging.INFO`. If that is the default we would need to add back a verbosity option.